pkg/cmd/task/describe.go

```diff
@@ -41,6 +41,10 @@ const describeTemplate = `{{decorate "bold" "Name"}}:	{{ .Task.Name }}
 {{decorate "bold" "Description"}}:	{{ .Task.Spec.Description }}
 {{- end }}
 
+{{- if ne (formatVersion .Task.Labels) ""}}
```

Comment:
Should we just get the value once, and use it both places, inspite of calling the func twice